services/platform/platform.go

```diff
@@ -311,3 +311,51 @@ func (s *Service) disconnect(ctx context.Context, params *disconnectPMMParams) e
 
 	return nil
 }
+
+// SearchOrganizationTickets fetches the list of ticket associated with the Portal organization this PMM server is registered with.
+func (s *Service) SearchOrganizationTickets(ctx context.Context, req *platformpb.SearchOrganizationTicketsRequest) (*platformpb.SearchOrganizationTicketsResponse, error) {
+	userAccessToken, err := s.grafanaClient.GetCurrentUserAccessToken(ctx)
+	if err != nil {
+		if errors.Is(err, services.ErrFailedToGetToken) {
+			return nil, status.Error(codes.FailedPrecondition, "Failed to get access token. Please sign in using your Percona Account.")
+		}
+		s.l.Errorf("SearchOrganizationTickets request failed: %s", err)
+		return nil, internalServerError
+	}
+	// Since PMM doesn't store the orgID we leave it empty and let Portal figure it out
+	// using the perconaPortal.orgID claim in the access token.
+	endpoint := fmt.Sprintf("https://%s/v1/orgs/%s/tickets:search", s.host, "")
```

Comment:
here we can use the following trick:
```suggestion
	endpoint := fmt.Sprintf("https://%s/v1/orgs/my/tickets:search", s.host)
```

and adjust Portal side to handle it properly.

Reply:
done portal changes are here https://github.com/percona-platform/orgd/pull/89/files#diff-201ce0a864ff7e42e2851e5300a344f1591e2f99bf2a8f05e8107b175a63733dR728

Reply:
regarding https://github.com/percona-platform/orgd/pull/89/files
there are changes in the specs accordingly to https://jira.percona.com/browse/PMM-9319?focusedCommentId=294004&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comm[…]4004
In short: now PMM Server knows organisation ID it is being connected to. So that it can use this ID in requests to Portal (“search tickets” in particular)

Sorry for inconvenience